GsaGH/Parameters/5_Results/6_Quantities/InternalForce.cs

```diff
@@ -8,11 +8,11 @@ public class InternalForce : IInternalForce {
     public Force X { get; internal set; }
     public Force Y { get; internal set; }
     public Force Z { get; internal set; }
-    public Force Xyz { get; internal set; }
+    public Force Xyz { get; internal set; } // here Yz!
```

Comment:
do we need this comments?

Reply:
yes, i wasn´t aware that Xyz is actually used for Yz